PatternPal/PatternPal.Extension/ViewModels/StepByStepListViewModel.cs

```diff
@@ -151,46 +151,44 @@ public class StepByStepListViewModel : ViewModel
                 {
                     string filePath;
 
-                    // The solution is empty.
-                    if (dte.Documents.Count == 0)
-                    {
-                        // Create a project based on a template from VS2022 add it to the 
-                        // solution. Then iterate over the projects of the solution and 
-                        // add a file.
-                        string projectName = "NewProject";
-                        string projectPath = Path.Combine(
-                            Path.GetDirectoryName(dte.Solution.FullName),
-                            projectName);
-
-                        // Obtain the template path for a project.
-                        DTE2 dte2 = (DTE2)Package.GetGlobalService(typeof(SDTE));
-                        Solution2 soln = dte2.Solution as Solution2;
-                        string templatePath = soln.GetProjectTemplate("ConsoleApplication.zip", "CSharp");
-
-                        // Add project to the solution and add a new file to that project.
-                        dte.Solution.AddFromTemplate(
-                            templatePath,
-                            projectPath,
-                            projectName,
-                            false);
-                        Project project = dte.Solution.Projects.Item(1);
-                        filePath =
-                            Path.Combine(
-                                projectPath,
-                                "NewFile.cs");
-                        using (FileStream fs = File.Create(filePath)) { }
-                        project.ProjectItems.AddFromFile(filePath);
-                    }
-                    else
+                    // Create a project based on a template from VS2022 add it to the 
+                    // solution. Then iterate over the projects of the solution and 
+                    // add a file.
+                    string projectName = SelectedInstructionSet.ToString();
+
+                    foreach (Project projectnames in dte.Solution.Projects)
```

Comment:
`projectNames` should be the casing here, but I think it is better to call the variable `project` since you are now calling `projectnames.Name` in the next line.